tests/test_registry.py

```
@@ -223,27 +232,63 @@ def test_func():
         self.assertEqual(r.by_label_contains("foo"), {"foo": [test_func]})
         self.assertEqual(r.by_label_contains("food"), {})
 
-    @mock.patch("side_effects.registry._run_func")
```

This test has been removed (not rewritten). It had hidden failures, and was incorrect: `no_return_value` _does_ actually receive a `return_value` in `kwargz`.

(The `test__run_side_effects__no_return_value` test failed silently because `_run_func` was mocked, but even after removing that, the assertion `assert "return_value" not in kwargz` actually _failed_. This failure would not be detected, though, because `settings.ABORT_ON_FAILURE` is off by default.)